src/tools/kata-ctl/src/arch/s390x/mod.rs

```diff
@@ -11,7 +11,7 @@ mod arch_specific {
     use crate::check;
     use anyhow::{anyhow, Result};
 
-    const PROC_CPUINFO: &str = "/proc/cpuinfo";
+    pub const PROC_CPUINFO: &str = "/proc/cpuinfo";
```

Comment:
Nit: How about moving this to `check.rs` and referencing it from all the arch-specific files?

Reply:
Thanks @jodh-intel. Looks like that filename doesnt need to be in the architecture files. I have moved it to check.rs